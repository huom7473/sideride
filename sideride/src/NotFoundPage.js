import logo from "./logo.svg";
import React from "react";

export default function NotFoundPage() {
    return (
        <div className="NotFoundPage">
            <header className="NotFoundPage-header">
                <img src={logo} className="App-logo" alt="logo" />
                <div>
                    404
                </div>
            </header>
        </div>
    );
}
