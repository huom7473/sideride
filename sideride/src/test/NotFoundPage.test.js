import NotFoundPage from "../NotFoundPage"
import { configure, shallow } from 'enzyme';
import Adapter from '@wojtekmaj/enzyme-adapter-react-17';
import Amplify from 'aws-amplify';
import awsconfig from '../aws-exports';


Amplify.configure(awsconfig);
// Auth.signIn("test_user", "test_password");

configure({ adapter: new Adapter() });      // for enzyme methods like mount

test('CheckboxWithLabel changes the text after click', () => {
  const wrapper = shallow(<NotFoundPage />);
  expect(wrapper).toMatchSnapshot();
});

