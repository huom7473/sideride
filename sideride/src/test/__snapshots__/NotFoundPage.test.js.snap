// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NotFoundPage matches snapshot 1`] = `
<div
  className="NotFoundPage"
>
  <header
    className="NotFoundPage-header"
  >
    <img
      alt="logo"
      className="App-logo"
      src="logo.svg"
    />
    <div>
      404
    </div>
  </header>
</div>
`;
